pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3588,13 +3588,14 @@
     },
     "System.Runtime.CompilerServices.Unsafe": {
       "StableVersions": [
-        "4.4.0"
+        "4.4.0",
+        "4.5.1"
```

Comment:
Can be addressed in a follow up, but typically we list all versions that have shipped, and for AssemblyVersionInPackageVersion we list the first version which contained the assembly version.  This enables us to produce minimum dependencies (if we should choose to).